Makefile

```diff
@@ -21,7 +21,7 @@ frontend:
 	yarn build
 
 backend:
-	python -m pip install requirements.txt
+	python -m pip install -r requirements.txt
```

Comment:
This is merged in another PR, I think you have to rebase.

Reply:
Will do once I refresh my memory on how. Re-learning some git skills.